packages/core/plugin/shorthandTerms.ts

```diff
@@ -0,0 +1,12 @@
+import type { KopflosPlugin } from '../lib/Kopflos.js'
+
+export default function (): KopflosPlugin {
```

Comment:
Although not specifically related to this PR, now I'm noticing the lack of expressivity of the plugin interface: given a Koplos instance, it does something with it. We may examine actual and potential  plugins and try to refine this interface

Reply:
What you mean is that giving free access to the dataset is too much and instead if should be something like

```ts
interface KopflosPlugin {
  expandApiGraph(): Iterable<Quad> | Stream
}
```

So that a plugin can insert into the API without being able to modify?